pkgs/tools/package-management/nix/default.nix

```diff
@@ -232,15 +232,21 @@ in rec {
   });
 
   nixUnstable = lib.lowPrio (callPackage common rec {
+    # Update checklist
+    #  - version
+    #  - date
+    #  - src rev/sha256
+    #  - changes in upstream packaging
+    #    git diff OLD..NEW -- $(find . -name \*.nix | grep -v test)
     pname = "nix";
     version = "2.4${suffix}";
-    suffix = "pre20210802_47e96bb";
+    suffix = "pre20210831_${lib.substring 0 7 src.rev}";
 
     src = fetchFromGitHub {
-      owner = "NixOS";
+      owner = "hercules-ci";
```

Comment:
Perhaps just a leftover from testing ?

Reply:
It's an unmerged PR. This is actually a draft PR now.